migrations/0036_order_time.up.sql

```diff
@@ -0,0 +1,9 @@
+alter table orders add column valid_for text default null;
+alter table orders add column expires_at timestamp with time zone default null;
+alter table order_items add column valid_for text default null;
+
+create table order_renewal (
```

Comment:
```suggestion
create table order_renewals (
```

Reply:
Wow this led me down a rabbit-hole of whether or not tables should be plural or singular (I've always assumes plural). Anyways, fine with either.

Reply:
i'm thinking we should change this table to be "order_payments" instead, as we are tracking payments not really renewals anymore.  thoughts?

Reply:
https://github.com/brave-intl/bat-go/pull/894/commits/012d4f0859d205d3c0abc772f8aa0421163b4cc0

Reply:
order_payment_history: https://github.com/brave-intl/bat-go/pull/894/commits/10be0fd563790266a216d97e70d87389d4b146d4